tests/e2e/functional_test.go

```diff
@@ -277,6 +324,12 @@ var _ = Describe("Functional e2e", Serial, func() {
 			return d.Spec.Template.Spec.Containers[0].Image == "quay.io/numaio/numaflow-rc:v0.0.13"
 		})
 
+		document("Verifying that the NumaflowControllerRollout conditions are as expected")
+		Eventually(func() metav1.ConditionStatus {
+			rollout, _ := numaflowControllerRolloutClient.Get(ctx, numaflowControllerRolloutName, metav1.GetOptions{})
+			return getRolloutCondition(rollout.Status.Conditions, apiv1.ConditionChildResourceHealthy)
+		}).WithTimeout(testTimeout).Should(Equal(metav1.ConditionTrue))
+
```

Comment:
how about the `Condition.PipelinesPausingOrPaused` for this and ISBSvc?

Reply:
Ack, resigned those in my head to just the pausing test. Will add those in all these checks.

Reply:
Thanks. I didn't realize you had to use `ConditionUnknown`. But I guess that's the only thing that works for both feature flag on and feature flag off? With feature flag, it would only be unknown briefly and then hopefully switch to `ConditionTrue` and then `ConditionFalse`. If so, maybe you were right to postpone that for when you have the environment variable ready after all? (sorry, I didn't recognize this when I asked)

Reply:
I think checking `ConditionUnknown` is not very useful right now...

Reply:
That's right, I have to check for `Unknown` at this time as PausingPipelines condition isn't present unless the flag is up. As for the environment variable, I could do that in a follow up PR or continue to add to this initial PR.

Reply:
follow up PR seems fine!

Reply:
Awesome, will just turn these current checks for PausingPipelines into comments for now.